lib/commons/builder/wikidata.rb

```diff
@@ -1,9 +1,23 @@
+require 'rest-client'
+
 class Wikidata
 
   attr_accessor :language_map
 
-  def initialize(language_map)
+  def initialize(language_map, url: 'https://query.wikidata.org/sparql')
     @language_map = language_map
+    @url = url
+  end
+
+  def perform(sparql_query)
+    query_params = {
+        query: sparql_query,
+        format: 'json',
+    }
+
+    result = RestClient.post(@url, params: query_params)
```

Comment:
So yes, I switched it to POST without actually running it again (mea culpa), because it /should/ work (https://www.w3.org/TR/sparql11-protocol/#query-via-post-urlencoded). And indeed, `curl -XPOST https://query.wikidata.org/sparql -dquery='ASK { ?s ?p ?o }'  -H"Accept: application/sparql-results+json" -v` does the right thing. I'll investigate and get this fixed.

Reply:
This is now fixed in 33b0f1b.